src/frontend/src/flows/manage/index.ts

```diff
@@ -453,3 +454,15 @@ export const domainWarning = (
 const unknownError = (): Error => {
   return new Error("Unknown error");
 };
+
+// Return a shuffled version of the array. Adapted from
+// https://stackoverflow.com/questions/2450954/how-to-randomize-shuffle-a-javascript-array
```

Comment:
can you update the link to point to the exact answer used?

Reply:
thought I did! my bad